### ISSUE TITLE
Missing release notes for xamarin android 11.1.0.26 and 11.2.0.0 and 11.2.0.21

### ISSUE THREAD

**Issue by tranb3r** [xamarin/xamarin-android#5498]
Visual studio 16.8.4 was published a few days ago.
Could you please publish release notes for xamain android 11.1.0.26.
cc @brendanzagaeski

**Comment by tranb3r** [comment]
cc @jonpryor

**Comment by jpobst** [comment]
I suspect there were no changes to Xamarin.Android, but @jonpryor should be able to confirm.

**Comment by tranb3r** [comment]
Previous version (in VS 16.8.3) was 11.1.0.17. If there is no change, why did you increase the version ?

**Comment by MagicAndre1981** [comment]
> Previous version (in VS 16.8.3) was 11.1.0.17. 

Check changes with this link:

https://github.com/xamarin/xamarin-android/compare/v11.1.0.17...v11.1.0.26

**Comment by tranb3r** [comment]
11.2.0.0 is out, and again no release notes.

**Comment by MagicAndre1981** [comment]
The user  @brendanzagaeski left the Xamarin.Android team from what I see the. So there will be no release notes until someone else does it.

Here is link with changes from Xamarin.Android between Preview 2 and Preview 3:

https://github.com/xamarin/xamarin-android/compare/v11.1.99.218...v11.2.0.0

**Comment by tranb3r** [comment]
@jonpryor  @jpobst Any update?

**Comment by tranb3r** [comment]
Sorry to tag you guys, but maybe someone in the team could publish those release notes ? The process seems to be broken...
@jonathanpeppers @pjcollins @radekdoulik @dellis1972 @marek-safar @brendanzagaeski 
Thank you !

**Comment by tranb3r** [comment]
New release 11.2.0.21

**Comment by brendanzagaeski** [comment]
Apologies for the slow reply on my part. MagicAndre1981 is correct that I'm on a different team now. So I'm no longer in the loop on the plans for how Xamarin.Android will be handling release notes moving forward.

MagicAndre1981's idea about using the version comparisons like <https://github.com/xamarin/xamarin-android/compare/v11.1.99.218...v11.2.0.0> should indeed be accurate, if a bit verbose. One refinement on that idea is that many of the changes add a descriptive Markdown file to the _Documentation/release-notes/_ directory. So a rough version of the release notes could be listed from command line Git with commands similar to:

```
git clone https://github.com/xamarin/xamarin-android.git
cd xamarin-android
git diff v11.1.99.218..v11.2.0.0 -- Documentation/release-notes
```

**Comment by tranb3r** [comment]
11.2.0.21 is now installed with visual studio 2019 stable.

**Comment by MagicAndre1981** [comment]
The Release Notes for 11.2 are [now published](https://docs.microsoft.com/en-us/xamarin/android/release-notes/11/11.2#march-2-2021--xamarinandroid-112021)

**Comment by tranb3r** [comment]
@MagicAndre1981 
11.2.2.1 is out (published in VS 2019 16.9.2). What's new ?

**Comment by MagicAndre1981** [comment]
> 11.2.2.1 is out (published in VS 2019 16.9.2). What's new ?

small [changes, 1 fast deployment fix](https://github.com/xamarin/xamarin-android/compare/v11.2.0.21...v11.2.2.1)
